docs/apidocs/qiskit-addon-aqc-tensor.rst

```diff
@@ -0,0 +1,12 @@
+#######################################################
+AQC Tensor root module (:mod:`qiskit_addon_aqc_tensor`)
```

Comment:
I wasn't sure what to call this page. We don't call it `root module` in other APIs, but imo something like this would be helpful. This h1 is used on the page itself & for the `<head>` used in link previews and search engines